src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle() {
+    }
+
+    public RightTriangle(double firstLeg, double secondLeg) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public RightTriangle(String name, String color, double firstLeg, double secondLeg) {
+        super(name, color);
```

Comment:
```suggestion
    public RightTriangle(String color, double firstLeg, double secondLeg) {
        super(color);
```
the name will be the same for each RightTriangle, just write RightTriangle in draw() method

this applies to each figure